src/components/List/List.jsx

```diff
@@ -3,23 +3,51 @@ import PropTypes from "prop-types";
 import cx from "classnames";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import "./List.scss";
+import { VirtualizedListItems } from "components/List/VirtualizedListItems/VirtualizedListItems";
+import { ListTitle } from "components";
 
-const List = forwardRef(({ className, id, component, children, dense, ariaLabel, ariaDescribedBy }, ref) => {
-  const componentRef = useRef(null);
-  const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
-  const Component = useMemo(() => component, [component]);
-  return (
-    <Component
-      ref={mergedRef}
-      className={cx("monday-style-list", className, { "monday-style-list--dense": dense })}
-      id={id}
-      aria-label={ariaLabel}
-      aria-describedby={ariaDescribedBy}
-    >
-      {children}
-    </Component>
-  );
-});
+const List = forwardRef(
+  ({ className, id, component, children, dense, ariaLabel, ariaDescribedBy, renderOnlyVisibleItems, style }, ref) => {
+    const componentRef = useRef(null);
+    const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
+    const Component = useMemo(() => component, [component]);
+    const overrideChildren = useMemo(() => {
+      let override = children;
+      if (Array.isArray(override) && override.length > 0) {
+        const originalFirstChildren = override[0];
+        if (originalFirstChildren.type === ListTitle) {
+          const firstChild = React.cloneElement(originalFirstChildren, {
+            ...originalFirstChildren?.props,
+            className: `${originalFirstChildren.className || ""} monday-style-list_category--first`
```

Comment:
all this just to add a class "first" - why not use a selector ":first" (also check it is a ListTitle)